packages/component/src/Attachment/Text/private/isAIGeneratedActivity.ts

```diff
@@ -0,0 +1,5 @@
+import { getOrgSchemaMessage, type WebChatActivity } from 'botframework-webchat-core';
+
+export default function isAIGeneratedActivity(activity: undefined | WebChatActivity) {
+  return !!(activity && getOrgSchemaMessage(activity?.entities || [])?.keywords?.includes('AIGeneratedContent'));
```

Comment:
In another universe, I am thinking what-if we do these kind of checks like...

```ts
return safeParse(
  array(
    object(),
    includes({
      '@id': '',
      '@type': 'Message',
      keywords: ...
    })
  ),
  activity.entities
).success;
```

Another universe...